theme.json

```diff
@@ -15,9 +84,97 @@
 					"name": "System fonts"
 				},
 				{
-					"fontFamily": "Helvetica Neue, Helvetica, Arial, sans-serif",
-					"slug": "helvetica-arial",
-					"name": "Helvetica or Arial"
+					"fontFamily": "\"Inter\", sans-serif",
+					"name": "Inter",
+					"slug": "inter",
+					"fontFace": [
+						{
+							"fontFamily": "Inter",
+							"fontStretch": "normal",
+							"fontStyle": "normal",
+							"fontWeight": "200 900",
+							"src": [
+								"file:./src/fonts/inter/Inter-VariableFont_slnt,wght.ttf"
+							]
+						}
+					]
+				},
+				{
+					"fontFamily": "Oxygen",
+					"slug": "oxygen",
+					"fontFace": [
+						{
+							"fontFamily": "Oxygen",
+							"fontStyle": "normal",
+							"fontWeight": "300",
+							"src": [
+								"file:./src/fonts/oxygen/Oxygen-Light.ttf"
+							]
+						},
+						{
+							"fontFamily": "Oxygen",
+							"fontStyle": "normal",
+							"fontWeight": "400",
+							"src": [
+								"file:./src/fonts/oxygen/Oxygen-Regular.ttf"
+							]
+						},
+						{
+							"fontFamily": "Oxygen",
+							"fontStyle": "normal",
+							"fontWeight": "900",
+							"src": [
+								"file:./src/fonts/oxygen/Oxygen-Bold.ttf"
+							]
+						}
+					]
+				}
+			],
+			"fontSizes": [
+				{
+					"fluid": {
+						"min": "0.75rem",
+						"max": "0.875rem"
+					},
+					"size": "0.875",
+					"slug": "small"
+				},
+				{
+					"fluid": {
+						"min": "0.875rem",
+						"max": "1rem"
+					},
+					"size": "1rem",
+					"slug": "normal"
+				},
+				{
+					"fluid": {
+						"min": "1rem",
+						"max": "1.125rem"
+					},
+					"size": "1.125rem",
+					"slug": "medium"
+				},
+				{
+					"fluid": {
+						"min": "1.75rem",
+						"max": "1.875rem"
+					},
+					"size": "1.75rem",
+					"slug": "large"
+				},
+				{
+					"fluid": false,
+					"size": "2.25rem",
+					"slug": "x-large"
+				},
+				{
+					"fluid": {
+						"min": "4rem",
+						"max": "10rem"
+					},
+					"size": "10rem",
```

Comment:
@itsamoreh This is actually mirroring TwentyTwentyThree which works for our design system as well, name is optional for fontSizes but it would not hurt to add that in.

Reply:
Adjusted to the size of our design system -> 7rem